swift/Tasks/TasksApp.swift

```diff
@@ -2,9 +2,17 @@ import SwiftUI
 
 @main
 struct TasksApp: App {
+    let ditto = DittoManager.shared.ditto
+
     var body: some Scene {
         WindowGroup {
-            TasksListScreen()
+            TasksListScreen().task {
+                do {
+                    try await ditto.store.execute(query: "ALTER SYSTEM SET DQL_STRICT_MODE = false")
```

Comment:
@okdistribute @aaronleopold we should either describe why disabling strict mode is necessary here or link to the relevant section on our docs website.

```suggestion
                    // TODO: describe or add link for why this is needed.
                    try await ditto.store.execute(query: "ALTER SYSTEM SET DQL_STRICT_MODE = false")
```